libs/dal/src/repositories/job/job.schema.ts

```diff
@@ -116,6 +116,13 @@ const jobSchema = new Schema<JobDBModel>(
       type: Schema.Types.ObjectId,
       ref: 'Subscriber',
     },
+    _tenantId: {
+      type: Schema.Types.ObjectId,
+      ref: 'Tenant',
+    },
+    tenantIdentifier: {
+      type: Schema.Types.String,
+    },
```

Comment:
I will remove the _tenantId and leave only the tenantIdentifier.
I feel it does make sense to have the tenantIdentifier as part of the job entity, also for future use when we incorporate more of the tenants logic in the future.